lib/ansible/utils/display.py

```diff
@@ -86,15 +69,15 @@ def get_text_width(text):
     if not isinstance(text, text_type):
         raise TypeError('get_text_width requires text, not %s' % type(text))
 
-    if _LOCALE_INITIALIZATION_ERR:
+    if LOCALE_INITIALIZATION_ERR:
         Display().warning(
-            'An error occurred while calling ansible.utils.display.initialize_locale '
+            'An error occurred while calling ansible.utils.locale.initialize_locale '
             '(%s). This may result in incorrectly calculated text widths that can '
-            'cause Display to print incorrect line lengths' % _LOCALE_INITIALIZATION_ERR
+            'cause Display to print incorrect line lengths' % LOCALE_INITIALIZATION_ERR
         )
-    elif not _LOCALE_INITIALIZED:
+    elif not LOCALE_INITIALIZED:
         Display().warning(
-            'ansible.utils.display.initialize_locale has not been called, '
+            'ansible.utils.locale.initialize_locale has not been called, '
             'this may result in incorrectly calculated text widths that can '
             'cause Display to print incorrect line lengths'
         )
```

Comment:
Can we remove these checks now that failure to initialize the locale is an error at startup?

Reply:
I could. I left them because I know there are people calling the APIs since we do have it documented, although listed as unsupported. Thoughts?

Reply:
I don't think we should maintain checks like this which are only applicable to third-party API usage. If we do maintain them, we should expose the checks from `ansible.cli` as well -- and then include unit tests for all of them. 

However, since it's an unsupported use case, I think we're better off just removing them instead of keeping around untested code.

Reply:
I've removed these warnings.